docs/contracts.rst

```diff
@@ -1080,12 +1082,11 @@ error "Linearization of inheritance graph impossible".
 
 ::
 
-    // This will not compile
-
     pragma solidity ^0.4.0;
 
     contract X {}
     contract A is X {}
+    // This will not compile
```

Comment:
I would suggest leave a comment here: "This is the reason this won't compile"

Reply:
do you suggest to use a different trigger phrase? If I change this line, the tests will break.